app/web/template/certificate_provider_your_details.gohtml

```diff
@@ -1,13 +1,36 @@
 {{ template "page" . }}
 
-{{ define "pageTitle" }}{{ end }}
+{{ define "pageTitle" }}{{ tr .App "yourDetails" }}{{ end }}
 
 {{ define "main" }}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <p class="govuk-body">TODO, but proof it works:</p>
-      <p class="govuk-body">Donor is {{ .Lpa.You.FullName }}</p>
-      <p class="govuk-body">Certificate provider is {{ .Lpa.CertificateProvider.FullName }}</p>
+      <form novalidate method="post">
+        <div class="govuk-form-group">
+          <fieldset class="govuk-fieldset">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+              <h1 class="govuk-fieldset__heading">{{ tr .App "yourDetails" }}</h1>
+            </legend>
+
+            <h2 class="govuk-heading-m">{{ tr .App "donor" }}</h2>
+            <p class="govuk-body">{{ .Lpa.You.FullName }}</p>
+
+            <h2 class="govuk-heading-m">{{ tr .App "yourDetails" }}</h2>
+            <p class="govuk-body">{{ .Lpa.CertificateProviderUserData.FullName }}</p>
```

Comment:
I still don't think we should ever use the value from identity as their name, only as a source for matching logic. But could use `.Lpa.CertificateProvider.FullName` and same for `.DateOfBirth`, `.Email` and `.Mobile`?

Reply:
I'm thinking of instances where the Donor has got some name details wrong about the CP and then they've identified themselves but will still be presented with the incorrect name? I guess this comes into play more when we decide how CPs will request changes to LPA data and also what content/design want to end up showing on these templates.

I've purposefully collected the other details here following the comments on the skeleton - https://miro.com/app/board/uXjVPL-znGI=/?openComment=3458764545649811033

Reply:
Since the name we get from ID may not be the same as you'd want to present yourself I don't think it should be used. I think it was a mistake for me to put `.FullName` on `UserData`, instead it should only give you the raw data plus some operations for (fuzzily) matching it.

Reply:
I finally looked for a source https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1124033/Names_-_names_that_cannot_be_used_in_passports__V9_FOR_GOV.UK_publication_.pdf and page 24 of https://www.icao.int/publications/Documents/9303_p3_cons_en.pdf

Reply:
also fun 

> The personal details section of the application allows the customer to use up to 30
characters for their forenames and 30 characters for their surname, including spaces
between them.

we allow more than that

Reply:
and more https://deedpolloffice.com/change-name/restrictions-on-names, showing you can have a name which isn't equivalent to your "passport name", or even if you don't care about a passport you can break their rules

Reply:
and something else to maybe consider for the Your Details page

>  There is no law preventing you from being known by a single name, or mononym — that is, a first name only, with no surname — and HM Passport Office should accept such a name, although they may be more sceptical of your application.

Reply:
Interesting - you've won me over. I'll drop a follow-up ticket in to rework the `UserData` stuff.

It would be really useful if we could get some example responses from GDS on names that contain diacritical characters or accent marks so we definitely know what we're working with here.